source/11-extending/05-reserved-values-available-for-extending-reports.rst

```diff
@@ -34,7 +34,7 @@ COUNTER recognizes that there are some common extensions that report providers m
      - Mt. Laurel University
 
    * - Format
```

Comment:
I'm wondering if the Format extension is used at all? If not, maybe we should remove it?

Reply:
I'm not aware of it being in use but I think it's fine to leave as-is for now